src/main/homework8/phonebook/Record.java

```diff
@@ -0,0 +1,19 @@
+package main.homework8.phonebook;
+
+public class Record {
+    private final String name;
+    private final Number number;
```

Comment:
В теорії String краще тут, телефонний номер часто не лише з цифр буде складатись. 
і навіть якщо буде, то не буде потреби робити якісь арифметичні обчислення з ним

```suggestion
    private final String number;
```